packages/app-store/_appRegistry.ts

```diff
@@ -44,7 +44,7 @@ export async function getAppRegistry() {
     // Skip if app isn't installed
     /* This is now handled from the DB */
     // if (!app.installed) return apps;
-
+    app.createdAt = dbapp.createdAt.toISOString();
```

Comment:
i always thought we already collect "createdAt" for apps, no? @hariombalhara

Reply:
We were storing it, it's just that were not sending it to frontend.